Modelica/Media/Incompressible.mo

```diff
@@ -840,10 +840,10 @@ in terms of tables, functions or polynomial coefficients.
 The common meaning of <em>incompressible</em> is that properties like density
 and enthalpy are independent of pressure. Thus properties are conveniently
 described as functions of temperature, e.g., as polynomials density(T) and cp(T).
-However, enthalpy can not be independent of pressure since h = u + p/d. For liquids
-it is anyway
-common to neglect this dependence since for constant density the neglected term
-is (p - p0)/d, which in comparison with cp is very small for most liquids. For
+However, enthalpy and inner energy can not both be independent of pressure since h = u + p/d. 
+(Normally dh/dp&ge;0 and du/dp&le;0.) For liquids it is anyway common to neglect this dependence for 
```

Comment:
Are dh/dp and du/dp partial derivatives? If yes, maybe use ∂ instead of d, and also state what variable is held constant.

Reply:
I can add "when T is held constant".

I would say they are total derivatives, but for formulas I find the total and partial difference a bit artificial. 

If we write h=u+p/d, then the partial derivative ∂h/∂p should be 1/d, right? 
But if we write h(p,T)=u(p,T)+p/d(p,T); then ∂h/∂p should be 1/d+∂u/∂p-p/(d*d)*∂d/∂p; and that corresponds to the total derivative, right?, - and the result I intended.